secant/Utils/Zatoshi.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Zatoshi.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 26.05.2022.
+//
+
+import Foundation
+
+typealias ZEC = NSDecimalNumber
```

Comment:
I could be missing something. But in terms of UI, I don't think we will be using ZEC as a numeric representation after all, but as a Decimal String representation of Zatoshi.

I see three approaches that are more limited but probably less error prone as we..

A. make ZEC a `struct` containing a Zatoshi and providing functions to and from string representations.
B. make a `String` extension and a `Zatoshi` extension for string representations (from and to)
C. create a ZecString struct that assures a given set of valid characters (see https://github.com/zcash/secant-android-wallet/blob/e44e9016769450c4e312c25992ad73d3d123936e/sdk-ext-ui-lib/src/main/java/cash/z/ecc/sdk/ext/ui/model/ZecString.kt#L59)

Reply:
ZEC removed completely